lib/const.rb

```diff
@@ -1,113 +1,116 @@
+# frozen_string_literal: true
+
 # Defines common constants used in different parts of crew
 
 CREW_VERSION = '1.23.16'
 
 ARCH_ACTUAL = `uname -m`.chomp
 # This helps with virtualized builds on aarch64 machines
 # which report armv8l when linux32 is run.
-ARCH = ( ARCH_ACTUAL == 'armv8l' ) ? 'armv7l' : ARCH_ACTUAL
+ARCH = ARCH_ACTUAL == 'armv8l' ? 'armv7l' : ARCH_ACTUAL
 
 # Allow for edge case of i686 install on a x86_64 host before linux32 is
 # downloaded, e.g. in a docker container.
-ARCH_LIB = ( ARCH == 'x86_64' and Dir.exist?('/lib64') ) ? 'lib64' : 'lib'
+ARCH_LIB = ARCH == 'x86_64' && Dir.exist?('/lib64') ? 'lib64' : 'lib'
```

Comment:
It doesn't hurt to keep the parenthesis for clarity.

Reply:
As with most of these changes, these are automatically done by rubocop. Should we look into changing the config file to be less strict? Maybe something more focused on indentations and whitespace?